packages/notifications/src/content/Paragraph.js

```diff
@@ -18,7 +17,7 @@ const COMPONENT_ID = 'notifications.paragraph';
 const Paragraph = styled.div.attrs({
```

Comment:
Should this be a function call `styled.div.attrs props => {`?

Reply:
That's only necessary if we are dynamically changing the classes based on prop changes. This always has a static className so it's not necessary.